src/renderer/scss/results.scss

```diff
@@ -16,27 +16,29 @@
   margin: auto;
 }
 
-.table {
+.Results-Table {
   margin: auto;
-}
-
-thead {
-  background-color: colors.$sand;
-}
-
-th {
-  border: 5px solid;
   text-align: center;
-  font-weight: bold;
-  font-size: large;
+  vertical-align: middle;
 }
 
-tr {
-  border: 5px solid;
+.Results-Table tr {
+  border: 3px solid;
```

Comment:
Maybe only do the border on the bottom of a row or something? and then have the top border filled by the table class?